app/views/ems_container/_form.html.haml

```diff
@@ -63,6 +63,47 @@
                        "required"                    => "",
                        "selectpicker-for-select-tag" => "")
 
+    %h4
+      = _("Environment Values for SmartState Analysis pods (Not Required)")
+    .form-group{"ng-class" => "{'has-error': angularForm.container_no_proxy.$invalid}"}
+      %label.col-md-2.control-label{"for" => "container_no_proxy"}
+        = _('NO_PROXY')
```

Comment:
I think each variable can be more descriptive like "No proxy Settings (e.g. localhost,127.0.0.1)"